easylist_adult/adult_adservers.txt

```diff
@@ -127,6 +128,7 @@
 ||getiton.com^$third-party
 ||ggwcash.com^$third-party
 ||golderotica.com^$third-party
+||hentaimotel.com$third-party
```

Comment:
It would be better with `a.hentaimotel.com` otherwise you might end up blocking things you wouldn't block. and this domain is just yet another dumb WP template: https://mypdns.org/my-privacy-dns/porn-records/-/issues/1496